dropletutils-read-10x-counts.R

```diff
@@ -13,14 +13,47 @@ option_list = list(
     action = "store",
     default = NA,
     type = 'character',
-    help = "A comma-separated list containing one or more directory names, each corresponding to a 10X sample. Each directory should contain the 'matrix.mtx', 'genes.tsv' and 'barcodes.tsv' files generated by CellRanger. Alternatively, strings may contain a path to a HDF5 file in the sparse matrix format generated by 10X. These can be mixed with directory names when type='auto'."
+    help = "A comma-separated list containing one or more directory names, each
+            corresponding to a 10X sample. Each directory should contain the 'matrix.mtx',
+            'genes.tsv' and 'barcodes.tsv' files generated by CellRanger. 
+            Alternatively, strings may contain a path to a HDF5 file in the sparse
+            matrix format generated by 10X. These can be mixed with directory 
+            names when type='auto'."
   ),
   make_option(
-    c("-c", "--col-names"),
+    c("-n", "--col-names"),
```

Comment:
Please revert this to what it was, and use a different short argument for your new options. The idea is to leave the interface untouched for any outside code calling this script as before.